docs/monoDrive_home/Vehicle-Configuration.md

```diff
@@ -1,89 +1,213 @@
 
-## Vehicle
+## Change the vehicle's model
+<div class="img_container">
+  <video width=650px height=480px muted controls autoplay loop>
+    <source src="http://cdn.monodrive.io/readthedocs/vehicles.mp4" type="video/mp4">
+  </video>
+</div>
+**Current Blueprints available**   
+On the monoDrive Editor look for the Content Browser, from the folders select Vehicles. You will find the current models available.
+
+When selecting a model programmatically (i.e. using any client) you can assign the model to use for the ego vehicle to be random from the vehicle blueprints, you can find it on your Editor folder under:  
+`Config/ConfigRandomVehicleSettings.ini`   
+
+```
+[VehicleBlueprints]
+VehicleBlueprint=Blueprint'/Game/Vehicles/sedan_monoDrive_01.sedan_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/sedan_monoDrive_02.sedan_monoDrive_02_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/SUV_monoDrive_01.SUV_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/truck_monoDrive_01.truck_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/subcompact_monoDrive_01.subcompact_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/minivan_monoDrive_01.minivan_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/coupe_monoDrive_01.coupe_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/compact_monoDrive_01.compact_monoDrive_01_C'
+```
+**Note:** If you want to add your own vehicle model, add the path to the blueprint to this file.  
+
+## Change the vehicle's color
+<div class="img_container">
+    <video width=650px height=340px muted controls autoplay loop>
+        <source src="http://cdn.monodrive.io/readthedocs/vehicle_color.mp4" type="video/mp4">
+    </video>
+</div> 
+
+**Current colors available**   
+On the monoDrive Editor look for the Content Browser. From the folders select Vehicles and then select CarPaint, you will find the current colors availables for any car.
+
+```
+Carpaint_BabyBlue
+Carpaint_Beige
+Carpaint_Black
+Carpaint_Copper
+Carpaint_DarkGrey
+Carpaint_Gold
+Carpaint_LightBlue
+Carpaint_NavyBlue
+Carpaint_OliveGreen
+Carpaint_Red
+Carpaint_Silver
+Carpaint_White
+```
+When selecting a color programmatically (i.e. using any client) you can assign the color to use for the ego vehicle using the `vehicle.json` with the `body` tag. 
+```
+"body": {
+    "color": "Carpaint_White.Carpaint_White",
+    "type": "/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C"
+  }
+```
+## PID Speed Controllers
+### Follower PID
+Each vehicle has a pre-defined PID parameters to control how closely and accurate follow others vehicles. You can tune this parameters.
```

Comment:
Each vehicle has a pre-defined PID parameters to control how closely and accurately it follows others vehicles.

Reply:
This is similar to the sentence below, do we need this section?

Reply:
One is related to the distance between one car and the other (follower) and the other is related to its own desired speed (speed).